salesforce/my-conservation-life/force-app/main/default/lwc/displayProjects/displayProjects.js

```diff
@@ -0,0 +1,151 @@
+import { LightningElement, track, wire } from 'lwc';
+import { projects } from 'c/controllers';
+import { CurrentPageReference } from 'lightning/navigation';
+import { registerListener, unregisterAllListeners } from 'c/pubsub';
+
+
+const actions = [
+    { label: 'Edit', name: 'edit' }
+];
+
+const columns = [
+    {
+        label: 'Project ID',
+        fieldName: 'id',
+        type: 'number'
+    },
+    {
+        label: 'Sponsor ID',
+        fieldName: 'sponsor_id',
+        type: 'number'
+    },
+    {
+        label: 'Project Name',
+        fieldName: 'name',
+        type: 'string'
+    },
+    {
+        label: 'Project Description',
+        fieldName: 'description',
+        type: 'string'
+    },
+    {
+        type: 'action',
+        typeAttributes: {
+            rowActions: actions,
+            menuAlignment: 'right'
+        }
+    }
+];
+
+export default class DisplayProjects extends LightningElement {
+
+    @wire(CurrentPageReference) pageRef;
+
+    @track data = [];
+    @track columns = columns;
+    @track tableLoadingState = true;
+    @track showModal = false;
+    @track currentRecord;
+    @track isEditForm = false;
+    @track updateHasSuccess = false;
+    @track updateHasError = false;
+    hasRendered = false;
+    updatedProject = {'id': undefined, 'sponsor_id': undefined, 'name': undefined, 'description': undefined};
+
+    // Listen for when a project is created so that we can refresh the table
+    connectedCallback() {
+        registerListener('projectCreated', this.updateTableEventHandler, this);
+    }
+
+    disconnectedCallback() {
+        unregisterAllListeners(this);
+    }
+
+    renderedCallback() {
+        // rendered callback may be called multiple times and we do not 
+        // want to keep hitting the servers
+        if (!this.hasRendered)
+        {
```

Comment:
I would move the { to the previous line for consistent code style

Reply:
^